<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tobias Doerffel</author>
 <class>RemoteLogonDialog</class>
 <widget class="QDialog" name="RemoteLogonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote logon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../italc.qrc">
    <normaloff>:/resources/remotelogon_48.png</normaloff>:/resources/remotelogon_48.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../italc.qrc">:/resources/remotelogon_48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Remote logon</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>Use the fields below to enter the username, password and optional the domain name. These information will be used to logon the computer(s).</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout">
     <property name="horizontalSpacing">
      <number>16</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="userNameEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Domain</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="domainEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>domainEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../italc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoteLogonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoteLogonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
